### PR TITLE
Ticket2338 instron

### PR DESCRIPTION
**Pull request by Tom-Willemsen**
See https://github.com/ISISComputingGroup/IBEX/issues/2338

### CODE REVIEW

File: instronSup/devinstron.proto [hunk 1]
```diff
@@ -15,7 +25,7 @@ getWatchdogStatus {
 }
 
 disableWatchdog {
-	out "C904,0,0";
+    out "C904,0";
```
Comment (Tom-Willemsen) [inline instronSup/devinstron.proto:25]: On inspection of network traces this was the wrong command previously.

File: instronSup/devinstron.proto [hunk 2]
```diff
@@ -25,11 +35,11 @@ getControlMode {
 
 setControlMode {
 	out "P909,%d";
-	@init { getControlMode; }
 }
 
 setControlModeCom {
     out "P909,1";
+	disableWatchdog;
```
Comment (Tom-Willemsen) [inline instronSup/devinstron.proto:39]: This is necessary to prevent the rig from tripping the hydraulics.

File: instronSup/devinstron.proto [hunk 3]
```diff
@@ -1,4 +1,14 @@
-Terminator = CR LF;
+#InTerminator = "";
+#OutTerminator = ";";
```
Comment (AdrianPotter) [inline instronSup/devinstron.proto]: As discussed, please remove unused code

**Review by AdrianPotter — CHANGES_REQUESTED** [review]
Tiny bit of clear up

**Review by AdrianPotter — APPROVED** [review]

**Pull request merged**